apps/newrelic.js

```diff
@@ -0,0 +1,36 @@
+var yaml = require('js-yaml');
+var fs = require('fs');
+var path = require('path');
+
+var licenseKey;
+try {
+  var locals = yaml.safeLoad(fs.readFileSync(path.join(__dirname, '..', 'locals.yml')));
+  licenseKey = locals.new_relic_license_key;
+} catch (e) {
+  // meh...
+}
+
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config.defaults.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
```

Comment:
I'm not seeing how our existing newrelic js is included into apps. It would be nice if we could convince ourselves that this won't collide with however we include newrelic in production.